src/views/ApartmentApp/components/StudentApplication/index.js

```diff
@@ -163,7 +163,7 @@ const StudentApplication = ({ userProfile, authentication }) => {
     }
 
     if (applicant.Profile.fullName === null) {
-      applicant.Profile = user.formatName(applicant.Profile);
+      applicant.Profile.fullName = user.setFullname(applicant.Profile);
```

Comment:
According to the definition of `setFullname` in the user.js file, this function appears to return a modified profile object with the `fullName` property added to it, rather than returning just the `fullName` as you have used it here. If that is not working as intended, then perhaps there is an addition problem in `user.js` that requires attention

Reply:
Alternatively, we should build this call of `setFullname` (or the above suggestion) into that part of the code that loads applications from the API, since any `Profile` that comes from `user.getProfileInfo` will already have the `fullName` property set by the `user.js`, and it is only the versions of `Profile` that come from `housing.js` that will be missing the `fullName` property. 

So maybe we should instead do this by default in the `getApartmentApplication` of `services/housing.js`

Reply:
You are correct, `setFullname` does indeed return the entire profile, so the previous way was more correct (although that function wasn't exported before).

I think adding it by default in `getApartmentApplication` is the most robust solution, certainly. I was just trying to make this work locally.